content/en/docs/tasks/configure-pod-container/enforce-standards-webhook.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Enforce Pod Security Standards by Configuring the Pod Security Admission Webhook
+reviewers:
+- tallclair
+- liggitt
+content_type: task
+min-kubernetes-server-version: v1.22
```

Comment:
We can omit this if we know that all Kubernetes features that your cluster is expected to provide have been in Kubernetes since v1.17 (or earlier).

Setting it changes the page text to tell readers they do need a cluster that recent (via `{{% version-check %}}` shortcode, and I think for 
validating Admission Webhooks the support went GA quite a while back.

Reply:
Per [dynamic admission registration docs](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#prerequisites), it's stable as of v1.16 -- will update accordingly